pytest/lib/test.py

```diff
@@ -0,0 +1,6 @@
+
+
+from configured_logger import new_logger
+
+L = new_logger(outfile="/tmp/something.txt")
+L.info("womp")
```

Comment:
Was this file intended to be included? Same with `welp.txt`?

Reply:
Good catch. Accidentally committed these